git_machete/github.py

```diff
@@ -161,7 +168,7 @@ def create_pull_request(org: str, repo: str, head: str, base: str, title: str, d
     }
     prs: List[GitHubPullRequest] = derive_pull_requests(org, repo)
     to_load: GitHubPullRequest = GitHubPullRequest(1, 'user', base, head, '')
```

Comment:
Wow I've only seen that now &mdash; that's pretty unclean, to create a fake `GitHubPullRequest` object just so that it matches the interface of `check_pr_already_created`